### ISSUE TITLE
PCL support

### ISSUE THREAD

**Issue by Aragas** [PlasmaShaft/Craft.Net#5]
Whole Craft.Net should work with PCL. We will be able to run it on desktops, mobile platforms and consoles. Why not use the cross-platforms possibilities from C#? Would be really cool to run both server and client on android, for example. How fast it will work is not so important

**Comment by Hetal728** [comment]
That would be cool. Unfortunately, the work Nirad did has some build errors. I'll fix them after my exams, and then we can set started on this after I upload the core server (PlasmaShaft).

**Comment by Hetal728** [comment]
One thing I have a good interest in is Windows 10 (App store). Windows 10 will be an universal OS, and if we could make a GUI application for it to allow it to run on Win10 Phones (Possibly), Win10 desktops, and XBox One.

**Comment by Aragas** [comment]
Sounds good. I can help with porting and creating mobile versions.

**Comment by Hetal728** [comment]
Nice, I assume we're going to be using Xamarin, so we'll have to see how much of it will be compatible

**Comment by Aragas** [comment]
Yep. Have it, so, we only need to port it to PCL, which will be really funny. PCL don't have any build-in Tcp/Udp client. We'll create an interface to interact with. Same goes to file system. A loong ride.

**Comment by Hetal728** [comment]
This looks promising https://github.com/rdavisau/sockets-for-pcl

**Comment by Aragas** [comment]
Have tried, but was not very useful. I use a bit different system, maybe I'll do my own lib for network. Isn't so hard. Should work for server too.

**Comment by Hetal728** [comment]
We could try this for the file system. https://github.com/dsplaisted/pclstorage

**Comment by Aragas** [comment]
Yep. Using it too. BouncyCastle and Json supports PCL. If i remember right ionic doesn't support. About other libs can't say anything. And yes, fnbt. This will be funny too.
